packages/compiler/vitest.config.mts

```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from 'vitest/config';
 import { resolve } from 'path';
+import { EXCLUDED_FILES_TEST } from '../../testing/setup/src/index.ts';
```

Comment:
This is an ugly way of importing a const from testing/setup. 
Unfortunately I did not manage to find a better way.